EvalData/models/data_assessment.py

```diff
@@ -576,12 +576,11 @@ def get_hit_status_for_user(cls, user):
     def get_time_for_user(cls, user):
         results = cls.objects.filter(createdBy=user, activated=False, completed=True)
 
-        durations = []
+        timestamps = []
         for result in results:
-            duration = result.end_time - result.start_time
-            durations.append(duration)
+            timestamps.append((result.start_time, result.end_time))
 
-        return seconds_to_timedelta(sum(durations))
+        return seconds_to_timedelta(_compute_user_total_annotation_time(timestamps))
```

Comment:
This fixes total annotation time on user's dashboards, right? Thanks

Reply:
Yes. Otherwise there would be a disprepancy.